### PR TITLE
DSERV-320 Merging Regulatory Regions and QTL API endpoints

### PR DESCRIPTION
**Pull request by pedrohr**

### CODE REVIEW

File: src/routers/datatypeRouters/edges/variants_genes.ts [hunk 1]
```diff
@@ -94,76 +71,21 @@ async function conditionalSearch (input: paramsFormatType, type: string): Promis
...
-  .meta({ openapi: { method: 'GET', path: '/variants/{variant_id}/genes' } })
-  .input(z.object({ variant_id: z.string() }).merge(variantsEqtlQueryFormat))
+  .meta({ openapi: { method: 'GET', path: '/variants/genes' } })
+  .input(z.object({ variant_id: z.string().optional() }).merge(variantsQtlsQueryFormat))
   .output(z.array(eqtlFormat.merge(sqtlFormat)))
-  .query(async ({ input }) => await conditionalSearch(input, 'all'))
+  .query(async ({ input }) => await qtlSearch(input))
 
 const variantsFromGenes = publicProcedure
   .meta({ openapi: { method: 'GET', path: '/genes/{gene_id}/variants' } })
```
Comment (mingjiecn) [inline src/routers/datatypeRouters/edges/variants_genes.ts:83]: `path: '/genes/{gene_id}/variants' ` should be `path: '/variants/genes' `?

File: src/routers/datatypeRouters/edges/variants_genes.ts [hunk 2]
```diff
@@ -94,76 +71,21 @@ async function conditionalSearch (input: paramsFormatType, type: string): Promis
...
-  .input(z.object({ gene_id: z.string() }).merge(variantsEqtlQueryFormat))
+  .input(z.object({ gene_id: z.string() }).merge(variantsQtlsQueryFormat))
   .output(z.array(eqtlFormat))
```
Comment (mingjiecn) [inline src/routers/datatypeRouters/edges/variants_genes.ts:85]: Should output merge `sqtlFormat` as well?

**Review by mingjiecn — CHANGES_REQUESTED** [review]
Please check my comment. Thank you!

File: src/routers/datatypeRouters/edges/variants_genes.ts (hunk 1 above)
Comment (pedrohr) [reply src/routers/datatypeRouters/edges/variants_genes.ts:83]: It should be /genes/variants, this was addressed in the genes PR

File: src/routers/datatypeRouters/edges/variants_genes.ts (hunk 2 above)
Comment (pedrohr) [reply src/routers/datatypeRouters/edges/variants_genes.ts:85]: `variantsQtlsQueryFormat` is the combination of all fields in eqtls and sqtls

**Review by mingjiecn — APPROVED** [review]

**Pull request merged**